syncmap_test.go

```diff
@@ -0,0 +1,134 @@
+package dapper_test
+
+import (
+	"sync"
+	"testing"
+)
+
+type syncmaps struct {
+	Map   sync.Map
+	Force bool
```

Comment:
I've been putting this comment elsewhere just as reminder.

```suggestion
	Force bool // prevent rendering of the zero-value marker
```